game.rb

```diff
@@ -0,0 +1,24 @@
+require_relative './item'
+require 'date'
+
+class Game < Item
+  attr_accessor :multiplayer, :last_played_at
+
+  def initialize(game_info, multiplayer, last_played_at)
+    super(game_info.genre, game_info.author, game_info.source, game_info.label, game_info.publish_date)
+    @multiplayer = multiplayer
+    @last_played_at = Date.parse(last_played_at)
+  end
+
+  def can_be_archived?
+    current_year = DateTime.now.year
+    age = current_year - @last_played_at.year
+    return true if super && age > 2
+
+    false
+  end
```

Comment:
- [ ] Again, this is not quite accurate since you're comparing only the years, please consider comparing the full date instead! 🤓 🔧 

```suggestion
    def can_be_archived?
      today = Date.today
      two_years_ago = Date.new(today.year - 2, today.month, today.day)

      super && @last_played < two_years_ago
    end

```